llama.h

```diff
@@ -358,6 +358,7 @@ extern "C" {
 
     //optional:
     LLAMA_API void llama_numa_init(enum ggml_numa_strategy numa);
+    LLAMA_API void llama_rpc_init(const char * rpc_servers);
```

Comment:
I don't think we should add this here. A less intrusive solution would be to pass the list of servers in `llama_model_params`, and have `ggml_backend_rpc_init` take the address of the server to connect to. The functions `llama_default_buffer_type_offload`, `llama_get_device_memory`, etc can be modified to take a `llama_model`, from which it can obtain the address of each server.

Reply:
I have added the list of RPC servers to both `llama_model_params` and `llama_model`